public_html/css/style.css

```diff
@@ -524,27 +524,17 @@ table.email-inbox {
     margin: 1em;
 }
 
-.advanced_search_modal {
-    z-index: 0;
-    /* Sit on top */
-    left: 0em;
-    top: 0em;
-    width: 60%;
-    /* Full width */
-    height: 100%;
-    /* Full height */
-    overflow: auto;
-    /* Enable scroll if needed */
-    background-color: rgb(0, 0, 0);
-    /* Fallback color */
-    background-color: rgba(0, 0, 0, 0);
-    /* Black w/ opacity */
-}
+.advanced_search_modal {}
```

Comment:
I would stick a comment here saying somthing along the lines of `/* TODO: Custom styling for Advanced Search Modal */`